### DISCUSSION TITLE
리트코드 스터디 5주차 모임

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#440] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. Sam입니다👋

다섯 번째 모임도 전 모임과 같이 지원자를 받고 각각 문제에 대해서 설명 및 논의하는 시간을 가지겠습니다!

문제당 한 분씩 풀이를 할 예정이고, 신청은 선착순으로 각 주차에 있는 아직 풀이 희망자가 없는 문제 중 원하시는 문제와 디스코드 명을 one-bit-integer / Sam 과 같은 형식으로 댓글로 남겨주시면 됩니다!ㅎㅎ

다음 모임도 이전과 마찬가지로 아래 링크로 RSVP 부탁드리겠습니다! (https://discord.gg/MfHvDEzV?event=1282361982527340545)

**Comment by DaleSeo** [comment]
@obzva 님, 지난 주 Word Search 문제 시간 복잡도 심층 분석해주실꺼죠? 😉
시간 복잡도가 `O(m * n * 4^w)`이 아닌 `O(m * n * 3^w)`가 되는 이유를 제대로 이해하고 싶습니다.

**Reply by obzva** [reply]
네 big-o에 대한 설명과 엮어서 준비해보겠습니다 ㅎㅎ

**Reply by TonyKim9401** [reply]
지난번 제 설명과 분석이 부족한걸 커버해주셔서 너무 감사합니다! 👍

**Reply by obzva** [reply]
ㅎㅎ 토니님도 함께 디스커션에서 이야기 나눠보아요

**Comment by TonyKim9401** [comment]
Implement Trie Prefix Tree / Tony

**Reply by TonyKim9401** [reply]
https://docs.google.com/presentation/d/1VqWpqt_rBwYh_Vzb5Qji-rL5ed3U6mrJgynxY0S3Yg0/edit?usp=sharing
이번주 발표자료 입니다.

**Comment by obzva** [comment]
word break / Flynn
